bridge/engine/SsrcOutboundContext.h

```diff
@@ -104,6 +105,11 @@ class SsrcOutboundContext
 
     // Stream owner is being removed. Stop outbound packets over this context
     bool markedForDeletion;
+
+    // This is for recording only! We will keep the SsrcOutbound context for while
```

Comment:
I see it is for recording by the variable name. The question is if the rest is bringing meaning to the reader.

// Retain rec OutboundSsrc before marking for deletion to sustain retransmissions longer.